mock-recipe-server/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
+
+services:
+  testgen:
+    extends:
+      file: normandy-base.yml
+      service: normandy-base
+    links:
+      - database
+      - autograph
+      - proxy
```

Comment:
I'm surprised you can reference link services from other files like this. It seems to work though. 👍